libc/docs/configure.rst

```diff
@@ -28,6 +28,13 @@ to learn about the defaults for your platform and target.
 * **"codegen" options**
     - ``LIBC_CONF_ENABLE_STRONG_STACK_PROTECTOR``: Enable -fstack-protector-strong to defend against stack smashing attack.
     - ``LIBC_CONF_KEEP_FRAME_POINTER``: Keep frame pointer in functions for better debugging experience.
+* **"errno" options**
+    - ``LIBC_CONF_ERRNO_MODE``: The implementation used for errno, acceptable values are:
```

Comment:
this file gets auto-generated, so the docs here need to be also in the json file

Reply:
I don't think JSON is particularly suited for multiline strings so I moved the explanation of different modes as comments to the implementation file.